teuthology/task/install/rpm.py

```diff
@@ -80,6 +84,32 @@ def _package_overrides(pkgs, os):
         result.append(pkg)
     return result
 
+def _zypper_addrepo(remote, repo_list):
+    """
+    Add packages to the remote system.
+
+    :param remote: remote node where to add packages
+    :param repo_list: list of dictionaries with keys 'name', 'url'
+    :return:
+    """
+    for repo in repo_list:
+        remote.run(args=[
+            'sudo', 'zypper', '-n', 'addrepo', '--refresh', '--no-gpgcheck',
+            '-p', '1', repo['url'], repo['name'],
```

Comment:
There is another instance of "addrepo" in teuthology/packaging.py - can you please add the options there, too.

Reply:
Yeah, I saw it. I was thinking it to reuse it, but it is not trivial.

Reply:
Yeah, we could just tolerate such a small amount of duplication.